chain/indexer/src/streamer/mod.rs

```diff
@@ -138,6 +147,11 @@ pub async fn build_streamer_message(
             {
                 debug_assert!(outcome.receipt.is_none());
                 outcome.receipt = Some(receipt.clone());
+            } else {
+                DELAYED_LOCAL_RECEIPTS_CACHE
```

Comment:
I want to prove the concept of having a cache. I am considering using the RocksDB instance we already use to keep track of the `last_synced_block_height` by the indexer. 🤔 Now I think the word `synced` should be replaced with `indexed`